src/components/CMSForm/index.tsx

```diff
@@ -82,7 +82,14 @@ export const CMSForm: React.FC<{
           setHasSubmitted(true)
 
           if (confirmationType === 'redirect' && redirect) {
-            if (redirect.url) router.push(redirect.url)
+            const { url } = redirect
+
+            const redirectUrl = new URL(url)
```

Comment:
Does this safely fail? Probably want to wrap with a condition if `url` exists then also a `try...catch`

Reply:
@jacobsfletch This has been added.